test/token_wrapper_test.ts

```diff
@@ -64,15 +64,15 @@ describe('TokenWrapper', () => {
             const toAddress = coinbase;
             return expect(zeroEx.token.transferAsync(
                 token.address, fromAddress, toAddress, transferAmount,
-            )).to.be.rejectedWith(ZeroExError.INSUFFICIENT_BALANCE_FOR_TRANSFER);
+            )).to.be.rejectedWith(ZeroExError.InsufficientBalanceForTransfer);
         });
-        it('should throw a CONTRACT_DOES_NOT_EXIST error for a non-existent token contract', async () => {
+        it('should throw a ContractDoesNotExist error for a non-existent token contract', async () => {
```

Comment:
We want to keep the caps version here because that is the error string that the user is going to receive.